tests/integration/test_interaction.py

```diff
@@ -271,6 +272,29 @@ def test_final_feedback_and_reset(self, action_key):
     def test_keyboard_help(self, use_keyboard):
         self.interact_with_keyboard_help(use_keyboard=use_keyboard)
 
+    def test_grade_display(self):
```

Comment:
@nasthagiri - taking over for @mtyaka on this temporarily.

We test for the grade event in tests/integration/test_events.py.